viewing_party/main.py

```diff
@@ -0,0 +1,151 @@
+#Wave 1 
+def create_movie(title,genre,rating): 
+    if (title and genre and rating): 
+        movie = {}
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    elif not title or not genre or not rating:
```

Comment:
This `elif` is just the opposite of the condition above so all you need here is an `else`:

```suggestion
    else:
```